pkgs/top-level/python-packages.nix

```diff
@@ -11585,6 +11585,8 @@ self: super: with self; {
     unicorn-emu = pkgs.unicorn;
   };
 
+  uni-curses = callPackage ../development/python-modules/uni-curses { };
```

Comment:
This does not match `pname` attribute in the derivation.

Reply:
```suggestion
  unicurses = callPackage ../development/python-modules/uni-curses { };
```